logpyle/__init__.py

```diff
@@ -919,7 +932,29 @@ def _commit(self) -> None:
             self.commit_countdown = self.commit_interval
             self.db_conn.commit()
 
+    def save_logging(self) -> None:
+        for log in self.logging_data:
+            self.db_conn.execute(
+                "insert into logging values (?,?,?,?,?,?,?)",
+                (self.rank, log.tick_count, log.time,
+                log.category, log.message, log.filename,
+                log.lineno))
+
+        self.logging_data = []
+
+    def save_warnings(self) -> None:
+        for w in self.warning_data:
+            self.db_conn.execute(
```

Comment:
It seems support for old schema versions was lost here.

Reply:
I can restore that support if you prefer, but is it useful to be able to _write_ older schema version?

Reply:
I'm not opposed to entirely *dropping* that support even, but there have to be error messages that clarify what happened, not failures/incorrect execution of `INSERT` somewhere in the guts.

Reply:
What do you think of 30ed3fb49f9e707ac506f8d5009a55fcad780686?

Reply:
Ok, I added this as issue #109.